merge User-Guides repository with meshcloud-docs repository (#529)
meshcloud/meshcloud-docs: docs/meshstack.how-to.manage-partner-level-permissions.md

---
id: meshstack.how-to.manage-partner-level-permissions
title: How to manage partner level permissions
---

If you are not familiar with what a Partner User is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/administration.index.html#docsNav).

## Pre-Requisites

- Permissions: Your user needs the Partner Admin role in the Partner Customer to assign other users administration and/or privileged roles

## Step to Step Guide

> 1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section but clicking on the "ADMIN AREA" button in the top navigation.

Reviewer: ```suggestion
1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section by clicking on the "ADMIN AREA" button in the top navigation.
```

![Select Admin Area in the top navigation bar](.././assets/partner/navigate-to-admin-area.png "Go to the Admin Area")

> 2. You now need to go to the Partner Access Control where you can to onboard further privieleged users.

Reviewer: ```suggestion
2. You now need to go to the Partner Access Control where you can onboard further privileged users.
```

![Select Partner Access Control menu item on the left side](../assets/partner/navigate-to-partner-access-control.png "Access Control")

> 3. At the end of the `Assigned Users & Groups` list will be a input field. Type in the first-, last-name or email address to find and select the user you want to add. Choose a Partner or Privielged Role and press the `+` button.

Reviewer: ```suggestion
3. At the end of the `Assigned Users & Groups` list will be an input field. Type in the first-, last name or email address to find and select the user you want to add. Choose a Partner or a privileged Role and press the `+` button.
```
